Add npm install to the README.md

```
[watch] build finished, watching for changes...
✘ [ERROR] Could not resolve "alpinejs"

    js/app.js:25:19:
      25 │ import Alpine from "alpinejs";
```

--- a/README.md
+++ b/README.md
@@ -3,6 +3,7 @@
 To start your Phoenix server:
 
 - Run `mix setup` to install and setup dependencies
+- Move to the assets folder with `cd assets` and run `npm install` to install the frontend dependencies.
 - Start Phoenix endpoint with `mix phx.server` or inside IEx with `iex -S mix phx.server`
 
 Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.

